dbms/src/Storages/AlterCommands.cpp

```diff
@@ -498,189 +504,95 @@ void AlterCommands::apply(StorageInMemoryMetadata & metadata) const
 }
 
 
-void AlterCommands::prepare(const StorageInMemoryMetadata & metadata, const Context & context)
+void AlterCommands::prepare(const StorageInMemoryMetadata & metadata, const Context & /*context*/)
```

Comment:
If it's not a virtual method or used in template - let's get rid of unused parameter completely.